Sources/RInAppMessaging/Repositories/CampaignRepository.swift

```diff
@@ -94,10 +85,10 @@ internal class CampaignRepository: CampaignRepositoryType {
                     updatedCampaign = Campaign.updatedCampaign(updatedCampaign, withImpressionLeft: newImpressionsLeft)
                 }
             }
-            updatedList.append(updatedCampaign)
+            return updatedCampaign
         }
-        campaigns.set(value: updatedList + testCampaigns)
-        saveDataToCache(updatedList + testCampaigns)
```

Comment:
The fix was about removing `testCampaigns`?

Reply:
The fix was actually about including test campaigns in the update closure above. So now, since test campaigns are basically treated as normal campaigns, there is no need to filter them out to a separate `testCampaigns` var